### PR TITLE
updated docs for cardanosharp

### PR DESCRIPTION
**Pull request by nothingalike**
#### Description of the change

CardanoSharp had some library updates and documentation updates. Just want to make sure developer portal is update to date :)

### CODE REVIEW

**Review by Mercurial — APPROVED** [review]

**Review by rdlrt — APPROVED** [review]

**Review by katomm — APPROVED** [review]
👍

**Review by Scitz0 — APPROVED** [review]
👍

File: docs/get-started/cardanosharp-wallet.md [hunk 1]
```diff
@@ -27,63 +19,70 @@ CardanoSharp.Wallet is installed from NuGet.
...
-        var addressService = new AddressService();
-        // 24 represents the number of words we want for our Mnemonic
-        int size = 24;
-
-        // This will generate a 24 English Mnemonic
-        Mnemonic mnemonic = keyService.Generate(size, WordLists.English);
-        System.Console.WriteLine(mnemonic.Words);
-    }
-}
+The `MnemonicService` has operations tbat help with *generating* and *restoring* Mnemonics. It is built for use in DI containers (ie. the interface `IMnemonicService`).
```
Comment (gufmar) [inline docs/get-started/cardanosharp-wallet.md:24]: minor typo "tbat"

File: docs/get-started/cardanosharp-wallet.md [hunk 2]
```diff
@@ -92,140 +91,226 @@ Address baseAddr = addressService.GetAddress(
...
+    .Derive(0);                     // IIndexNodeDerivation
+
+PrivateKey privateKey = derivation.PrivateKey;
+PublicKey publicKey = derivation.PublicKey;
 ```
 
 ## Build and Sign Transactions
 
-This is just an example of how to start. You will need to Calculate Fees, compare with Protocol Parameters and re-serialize.
+CardanoSharp.Wallet requires input from the chain in order to build transactions. Lets assume we have gathered the following information.
```
Comment (gufmar) [inline docs/get-started/cardanosharp-wallet.md:127]: another minor, and sorry, not really important but "let's assume" sounds more correct also in 4 other lines

**Pull request merged**